rtb-house/DOCUMENTATION.md

```diff
@@ -2,56 +2,106 @@
 ## General Compatibility
 |Feature|  |
 |---|---|
-| Consent |  |
-| Native Ad Support |  |
-| SafeFrame Support |  |
-| PMP Support | |
+| Consent | Yes |
+| Native Ad Support | No |
+| SafeFrame Support | No |
+| PMP Support | No |
  
 ## Browser Compatibility
 | Browser |  |
 |--- |---|
-| Chrome |  |
-| Edge |  |
-| Firefox |  |
-| Internet Explorer 9 |  |
-| Internet Explorer 10 |  |
-| Internet Explorer 11 |  |
-| Safari |  |
-| Mobile Chrome | |
-| Mobile Safari | |
-| UC Browser | |
-| Samsung Internet | |
-| Opera | |
+| Chrome | Yes |
+| Edge | Yes |
+| Firefox | Yes |
+| Internet Explorer 9 | Yes |
+| Internet Explorer 10 | Yes |
+| Internet Explorer 11 | Yes |
+| Safari | Yes |
+| Mobile Chrome | Yes |
+| Mobile Safari | Yes |
+| UC Browser | Yes |
+| Samsung Internet | Yes |
+| Opera | Yes |
  
 ## Adapter Information
 | Info | |
 |---|---|
 | Partner Id | RTBHouseHtb |
-| Ad Server Responds in (Cents, Dollars, etc) | |
-| Bid Type (Gross / Net) | |
-| GAM Key (Open Market) | |
-| GAM Key (Private Market) | |
-| Ad Server URLs | |
-| Slot Mapping Style (Size / Multiple Sizes / Slot) | |
-| Request Architecture (MRA / SRA) | |
+| Ad Server Responds in (Cents, Dollars, etc) | Dollars |
+| Bid Type (Gross / Net) | Net |
+| GAM Key (Open Market) | n/a |
+| GAM Key (Private Market) | n/a |
+| Ad Server URLs | https://prebid-[eu/asia/us].creativecdn.com/bidder/prebid/bids? |
+| Slot Mapping Style (Size / Multiple Sizes / Slot) | Multiple Sizes |
+| Request Architecture (MRA / SRA) | MRA |
  
 ## Currencies Supported
  
 ## Bid Request Information
 ### Parameters
 | Key | Required | Type | Description |
 |---|---|---|---|
-| | | | |
- 
+|id|true|string|Unique ID of the bid request, provided by the exchange|
+|imp|true|object array|Array of Imp objects|
+|site|false|object|Details via a Site object about the publisher’s website.|
```

Comment:
jakikolwiek identyfikator site'a i najlepiej slotu jest przez nas wymagany - inaczej będziemy bidować losowo

Reply:
Czyli my wymagamy, mimo że w OpenRTB jest to tylko "recommended"?

Reply:
a inni jak robią? Bez site pewnie coś zabidujemy, ale głupio

Reply:
Niektórzy wymagają, więc w sumie myśle że można - przestawiam.